pupil_src/shared_modules/recorder.py

```diff
@@ -189,7 +189,14 @@ def on_notify(self, notification):
             if self.running:
                 logger.info('Recording already running!')
             elif not self.g_pool.capture.online:
-                logger.error("Current world capture is offline. Please reconnect or switch to fake capture")
+                logger.warning("Current world capture is offline. Starting fake capture.")
+                settings = {}
+                settings['frame_rate'] = self.g_pool.capture.frame_rate
+                settings['frame_size'] = self.g_pool.capture.frame_size
+                settings['name'] = self.g_pool.capture.name
+                self.notify_all({'subject': 'start_plugin', "name": "Fake_Source", 'args': settings})
+                self.notify_all({'subject': 'recording.should_start',
```

Comment:
you a reacting to a recoding.should start with another recording.should start notification. has this be tested? What happens when used with pupil groups?

Reply:
I tested it only locally. The idea was to ensure that the recording would start after the fake capture had started. Resending the `recording.should_start` would trigger a `Recording already running!` info message on other pupil groups computers if these had started their recording previously. Any idea on how to fix this conflict? We cannot prevent other group members from receiving the first message.